changed.d

```diff
@@ -419,19 +424,14 @@ Please supply a bugzilla version
             changedRepos.each!(r => r.changes.writeTextChangesHeader(w, r.headline));
 
             if (!revRange.empty)
-            {
-                if (useNightlyTemplate)
-                    w.put("$(BR)$(BIG $(RELATIVE_LINK2 bugfix-list, List of all upcoming bug fixes and enhancements.))\n\n");
```

Comment:
I don't understand how this really simplifies generation, and I think saying "list of things in $VERSION" when $VERSION hasn't been released yet is a little bit worse?